dashboard/app/controllers/api/v1/census/census_controller.rb

```diff
@@ -1,3 +1,5 @@
+require 'cdo/email_preference_constants'
```

Comment:
Is there some way I can "publish" or "export" this so it doesn't have to be explicitly required in each Dashboard controller and Pegasus Form that needs to write to EmailPreference?